daemon/provider/musicvideoinfo/musicvideoinfoprovider.py

```diff
@@ -13,3 +13,6 @@ def get_albums_by_artist_id(artist_id):
 
     def get_tracks_by_album_id(album_id):
         raise NotImplementedError("get_tracks_by_album_id is not implemented.")
+
+    def get_new_release_by_album_id(offset_count,limit):
```

Comment:
We should get all new release albums one time. Hence, we don't need to give the `offset` and the `limit`.
So you should refactor this function
```
def get_new_release_albums():
    raise NotImplementedError("get_new_release_albums is not implemented.")
```